scripts/formats.js

```diff
@@ -1,33 +1,113 @@
+const fs = require('fs');
+const path = require('path');
+const SD_scssFormat = require('../node_modules/style-dictionary/lib/common/formats')['scss/variables'];
+
 /**
  * StyleDictionary custom formats
  * https://amzn.github.io/style-dictionary/formats_and_templates
  */
 
+// load and keep breakpoint values in memory
+// for responsive var value output
+const bpProperties = JSON.parse(
+		fs.readFileSync(
+			path.join(__dirname, '..', 'properties', 'layout', 'breakpoints.json')
+		)
+	);
+
+// get media query string from size keyword
+// (e.g. 's', 'm', 'l')
+const getMQBySize = (size) =>
+	`only screen and (min-width: ${bpProperties.layout.breakpoint[size].value})`;
+
+// generated content warning for top of files
 const dateHeader = () =>
 	  '/**\n' +
 		' * Do not edit directly\n' +
 		' * Generated on ' + new Date() + '\n' +
 		' */\n\n';
 
+
+// commonJS module format
+// (!) "color" category only
+//
+// adds every dictionary property to exports
 const commonJS = {
 	name: 'javascript/commonJS',
-	formatter: (dictionary, platform) => dateHeader() + dictionary
+	formatter: (dictionary) => dateHeader() + dictionary
 		.allProperties
+		.filter(prop => prop.attributes.category === "color")
```

Comment:
FYI - I had to adjust some existing formats to filter out non-color properties now that there are some non-colors in constants.